chromatic/.storybook/preview.ts

```diff
@@ -0,0 +1,77 @@
+import type { Preview } from '@storybook/web-components';
+import '@webcomponents/scoped-custom-element-registry/scoped-custom-element-registry.min.js';
+import 'element-internals-polyfill';
+import { configureLocalization } from '@lit/localize';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+import { updateTheme, themes } from '../../.storybook/themes';
+
+const { setLocale } = configureLocalization({
+  sourceLocale: 'en',
+  targetLocales: ['nl'],
+  loadLocale: locale => import(`../packages/locales/src/${locale}.ts`)
+});
+
+const preview: Preview = {
+  decorators: [
+    (story, { globals: { locale = 'en' } }) => {
+      document.documentElement.lang = locale;
+      setLocale(locale);
+
+      return story();
+    },
+    (story, { globals: { mode = 'light', theme = 'sanoma-learning' } }) => {
+      updateTheme(theme, mode);
+      
+      return story();
+    }
+  ],
+  globalTypes: {
+    theme: {
```

Comment:
Do we need these global types? For chromatic, we set the theme in the story, right? Idem with light/dark mode below this.